gcm_filters/filter.py

```diff
@@ -317,10 +327,23 @@ def __post_init__(self):
                     "dx_min must be set to 1."
                 )
 
+        # Check for n_iterations is < 1
+        if self.n_iterations < 1:
+            raise ValueError(
+                f"Number of filters into which the Gaussian is factored must be >= 1."
```

Comment:
This error message could be confusing if the user has specified a Taper filter shape.

Reply:
OK, I will drop 'Gaussian.' If they are using the Taper filter and setting `n_iterations` to anything other than 1, then it will raise a warning later.